src/azul/strings.py

```diff
@@ -20,3 +20,29 @@ def departition(before, sep, after):
         return before
     else:
         return before + sep + after
+
+
+def pluralize(word: str, count: int) -> str:
+    """
+    Appends 's' or 'es' to `word` following common patterns in English spelling
+    if `count` indicates that the word should be pluralized.
+
+    >>> pluralize('foo', 1)
+    'foo'
+
+    >>> pluralize('bar', 2)
+    'bars'
+
+    >>> pluralize('baz', 2)
+    'bazes'
+
+    >>> pluralize('woman', 2)
+    'womans'
```

Comment:
+1 for adding a test that demonstrates the limitations of the current implementation. In the future we may switch to https://inflection.readthedocs.io/en/latest/ if we think it's worth it.